README.md

```diff
@@ -21,8 +21,11 @@ Below is an example of how this package may be used. The options below are used
 package main
 
 import (
-    "os"
-	"github.com/Sirupsen/logrus"
+	"os"
+	"strings"
```

Comment:
Having implemented the changes below, do make sure to remove unused imports.

Reply:
✔